calebhsu/CoMake
# Updates production config for deployment

Closes #113 

* Switched from UglifyJS to Babili for ES2016 minification support
* Moved some .md files to Wiki

To test if working, 
` $ npm run build ` in the website/ directory
` $ firebase serve ` in the root directory

It took 9 minutes for me to build yesterday so if it hangs it should be ok.
